vllm/model_executor/layers/rotary_embedding.py

```diff
@@ -1096,7 +1096,7 @@ def get_input_positions_tensor(
                 audio_feature_lengths=audio_feature_lengths,
                 use_audio_in_video=use_audio_in_video,
             )
-        elif "glm4v" in hf_config.model_type:
+        elif "glm4v" or "glm4v_moe" in hf_config.model_type:
```

Comment:
```suggestion
        elif hf_config.model_type in ["glm4v", "glm4v_moe"]:
```
I think `hf_config.model_type` should be a string.